source/xep/httpFileUpload/DownloadManager.cpp

```diff
@@ -147,3 +189,58 @@ void DownloadManager::downloadFinished(QNetworkReply *reply)
     }
 }
 
+
+bool DownloadManager::aesDecrypt(const QByteArray &key, const QByteArray &iv, const QByteArray &in, QByteArray &out)
```

Comment:
Move this to CryptoHelper class as a static callable method.

Reply:
> Nice to hear that you will also add encryption of images.
> Do you talk about encrypted video calls? I did not look into the xep for now, but this will be a tough task. It also took me a while to 'just' implement 1to1 omemo.
> 
> For the filename stuff, this is just a minor change of some lines of code. I will do this by myself after your PR is merged. I have only a few comments on your code, I will write to the source.

I mean exchanging video files. For video calls, I'm not skilled enough (at least for the moment)